.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     docker:
       # CircleCI Go images available at: https://hub.docker.com/r/circleci/golang/
       - image: circleci/golang:1.13.1
-    resource_class: large
```

Comment:
I was getting repeated failed workflows with the following error
```
/usr/local/go/pkg/tool/linux_amd64/link: signal: killed
```

https://app.circleci.com/jobs/github/argoproj/argo-rollouts/651
https://app.circleci.com/jobs/github/argoproj/argo-rollouts/652
https://app.circleci.com/jobs/github/argoproj/argo-rollouts/650

This Stackoverflow suggested it was a resourcing issue: https://stackoverflow.com/questions/52129612/golang-circleci-2-0-test-failing-with-signal-killed